bunsen/pom.xml

```diff
@@ -90,6 +90,12 @@
       <artifactId>jackson-core</artifactId>
       <version>2.18.1</version>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
+      <version>4.5.0-M2</version>
```

Comment:
Just wanted to understand why we upgrading to a milestone version here?

Reply:
I think I used this version because 4.4 was released 5 years ago (according to [Maven](https://central.sonatype.com/artifact/org.apache.commons/commons-collections4/versions)) and there was no 4.5; or maybe it was because of other dependencies. Anyways, I changed this to `4.4` for now.